css/main.css

```diff
@@ -325,3 +325,28 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact {
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 30px;
+}
```

Comment:
- [ ] Please try to fix the alignment issue in the contact section. So that it gives a better visual experience. 

![image](https://user-images.githubusercontent.com/53019916/124177164-7cb01f00-dacd-11eb-98dc-2c0b127b34d8.png)

Reply:
Please make this change requested by the previous TSE regarding the alignment issues in the contact form.